package.json

```diff
@@ -21,11 +21,15 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-n": "^15.3.0",
     "eslint-plugin-promise": "^6.0.1",
-    "typescript": "^4.8.3"
+    "gulp-obfuscate": "^0.2.9",
+    "gulp-uglify": "^3.0.2",
+    "typescript": "^4.9.4"
   },
   "dependencies": {
     "axios": "^0.27.2",
     "crypto-js": "^4.1.1",
+    "gulp": "^4.0.2",
```

Comment:
gulp is not being used in app.ts. It should be in dev dependencies.

Reply:
Okay will update